addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/DataGenerator.java

```diff
@@ -203,6 +207,14 @@ public String getEnumValue(Schema<?> schema) {
         return value;
     }
 
+    public boolean hasExample(Schema<?> schema) {
+        return schema.getExample() != null;
+    }
+
+    public Object getExample(Schema<?> schema) {
```

Comment:
This seems unnecessary, the getter can be called directly like in `BodyGenerator`.